### PR TITLE
add bounce style

### PR DESCRIPTION
**Pull request by flipactual**
bounce

### CODE REVIEW

**Comment by arthurvr** [comment]
Thanks for the PR. No idea why it took that long for someone to reply, sorry therefore :) 

Tests are failing would you mind taking a look? @sindresorhus Do you like the idea or do you think it's out of scope?

File: index.js [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
-	['-', '\\', '|', '/'] :
-	['⠋', '⠙', '⠹', '⠸', '⠼', '⠴', '⠦', '⠧', '⠇', '⠏'];
+var styles = {
+  win32: ['-', '\\', '|', '/'],
+  spinner: ['⠋', '⠙', '⠹', '⠸', '⠼', '⠴', '⠦', '⠧', '⠇', '⠏'],
+  bounce: ['⠁', '⠂', '⠄', '⠂']
+};
 
-module.exports = function () {
+module.exports = function (style) {
```
Comment (kevva) [inline index.js]: I rather have an options object here in case we add additional options in the future.

Comment (SamVerschueren) [inline index.js]: :+1:

File: index.js [hunk 2]
```diff
@@ -12,4 +21,4 @@ module.exports = function () {
 	};
 };
 
-module.exports.frames = frames;
+module.exports.styles = styles;
```
Comment (SamVerschueren) [inline index.js]: Not sure about this change.

File: index.js [hunk 3]
```diff
@@ -1,10 +1,19 @@
...
-	['⠋', '⠙', '⠹', '⠸', '⠼', '⠴', '⠦', '⠧', '⠇', '⠏'];
+var styles = {
+  win32: ['-', '\\', '|', '/'],
+  spinner: ['⠋', '⠙', '⠹', '⠸', '⠼', '⠴', '⠦', '⠧', '⠇', '⠏'],
+  bounce: ['⠁', '⠂', '⠄', '⠂']
+};
 
-module.exports = function () {
+module.exports = function (style) {
+  var frames = styles[
```
Comment (kevva) [inline index.js]: This is cleaner in my opinion:

```js
var style = (process.platform === 'win32' ? 'win32' : opts.style) || 'spinner';
var frames = styles[style];
```

File: index.js (hunk 2 above)
Comment (kevva) [inline index.js]: Me neither. Also, this is why tests are failing I guess.

**Comment by flipactual** [comment]
closing in favor of #3

**Pull request closed without merging**